lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -267,6 +267,42 @@ def request_params_hosts(params)
       end
     end
 
+    class ExportDefaultCommand < HammerCLIForeman::Command
+      include HammerCLIKatello::LocalHelper
+      include HammerCLIKatello::ApipieHelper
+
+      PUBLISHED_REPOS_DIR = "/var/lib/pulp/published/yum/https/repos/".freeze
+
+      desc _('Export the library default content view')
+
+      command_name "export-default"
+
+      success_message _("Default content view export is available at")
+      failure_message _("Could not export the default content view at")
+
+      option '--export-dir', 'EXPORT_DIR', _("Directory to put content view version export into.")
+
+      validate_options do
+        option(:option_export_dir).required
+      end
+
+      build_options
+
+      def execute
+        export_dir = options['option_export_dir']
+
+        Dir.mkdir(export_dir) unless Dir.exist?(export_dir)
+        result = Kernel.system("rsync -aL #{PUBLISHED_REPOS_DIR} #{export_dir}")
+        if result == true
```

Comment:
This is not normal in Ruby, but I can understand why you must explicitly check for `true`. From the ruby docs:

> system returns true if the command gives zero exit status, false for non zero exit status. Returns nil if command execution fails

I think it will return `nil` only if the `rsync` command isn't found on the system, which is relevant.

Reply:
I agree I was having some issues with tests and it working correctly until I did it this way. Not the best way but sadly it will have to do for now until we push export into its own module and look at a better way to export etc.